src/ExpressReceiver.ts

```diff
@@ -216,63 +221,70 @@ export function verifySignatureAndParseBody(
 async function verifyRequestSignature(
   signingSecret: string,
   body: string,
-  signature: string,
-  requestTimestamp: number): Promise<void> {
-  if (!signature || !requestTimestamp) {
-    const error = errorWithCode(
+  signature: string | undefined,
+  requestTimestamp: string | undefined,
+): Promise<void> {
+  if (signature === undefined || requestTimestamp === undefined) {
+    throw errorWithCode(
       'Slack request signing verification failed. Some headers are missing.',
       ErrorCode.ExpressReceiverAuthenticityError,
     );
-    throw error;
+  }
+
+  const ts = Number(requestTimestamp);
```

Comment:
updated this function to do the conversion of the string timestamp, since that comes with its own issues/logic we need to handle. we _could_ do this outside of this function and before we call `verifyRequestSignature`, and then pass a `number` in instead of a `string`, but if we did we'd have to do that same logic and error handling every time before we called this function.

Reply:
the downside here is if anyone was importing `@slack/bolt/dist/ExpressReceiver` directly, and accessing `verifyRequestSignature()` in their code, this would break their code. i think that's okay because we don't explicitly export this function from `index`, so it's "at their own risk" if someone went down that path.